btcnode.adoc

```diff
@@ -0,0 +1,95 @@
+
+
+== Introduction
+
+In the past, Bisq connected to arbitrary Bitcoin Core nodes, but the prospect of the SegWit2x hard fork in Fall 2017 made it clear this was not ideal—in the case of a hard fork, it would be impossible for Bisq to tell which fork a connected node was running. The SegWit2x hard fork never happened, but if it did, there was a chance transactions would execute on an undesirable fork, and even worse, a chance that transactions would execute on different forks.
+
+Critically, bloom filters introduce serious privacy flaws when connecting to arbitrary bitcoin nodes, as chain analysis companies spy on SPV wallets. Connecting only to nodes run by trusted Bisq contributors avoids such monitoring.
+
+Hence the decision to run a federation of Bitcoin nodes: more robust user privacy and a consistent Bitcoin Core implementation Bisq can rely on.
+
+Note: Bisq users can still choose to connect to arbitrary public nodes, as before, or even connect to specific nodes they prefer.
+
+== Infrastructure
+
+=== GitHub
+
+Issues are managed in the {gh-org}/bisq-btcnode[bisq-network/bisq-btcnode] repository.
+
+=== Slack
+
+Discussion of Bitcoin Core node status and updates takes place in the `#bisq-btcnode` Slack channel.
+
+
+== Roles
+
+
+=== Maintainer
+
+The contributor(s) responsible for full node <<infrastructure>> and <<process>>.footnote:[See link:roles.html#maintainer[]]
+
+==== Role Issue
+
+{gh-org}/roles/issues/66[bisq-network/roles#66] footnote:[See link:roles.html#issue[]]
+
+==== Role Team
+:btcnode-maintainers: {gh-team}/btcnode-maintainers[@bisq-network/btcnode-maintainers]
+
+{btcnode-maintainers} footnote:[See link:roles.html#team[]]
+
+==== Duties
+
+ * Monitor communications on the `#bisq-btcnode` Slack channel.footnote:[See link:roles.html#communication[]]
+ * Keep this documentation up to date (in particular, the <<process>> detailed below).footnote:[See link:roles.html#documentation[]]
+ * Write a monthly report on the Bitcoin Node Maintainer <<role-issue>>.footnote:[See link:roles.html#reporting[]]
+
+==== Rights
+
+ * Write access to the {gh-org}/bisq-btcnode[bisq-network/bisq-btcnode] repository.
+
+=== Operator
+
+The contributor(s) responsible for keeping full nodes running as outlined in <<process>>.footnote:[See link:roles.html#maintainer[]]
+
+==== Role Issue
+
+{gh-org}/roles/issues/67[bisq-network/roles#67] footnote:[See link:roles.html#issue[]]
+
+==== Role Team
+:btcnode-operators: {gh-team}/btcnode-operators[@bisq-network/btcnode-operators]
+
+{btcnode-operators} footnote:[See link:roles.html#team[]]
+
+==== Duties
+
+ * Operate bitcoin nodes as detailed below in <<process>>.
+ * Monitor communications on the `#bisq-btcnode` Slack channel.footnote:[See link:roles.html#communication[]]
+ * Write a monthly report on the Bitcoin Node Operator <<role-issue>>.footnote:[See link:roles.html#reporting[]]
+
+==== Rights
+
+ * N/A
+
+
+== Process
```

Comment:
I'm not sure what the process to add more nodes should be, but like many of the other roles a requirement would be to already be in good standing within the Bisq community since it's a trusted role. It's not as sensitive as arbitrator or pricenode/seednode operator but it's also impossible to find out if an operator is breaking privacy in any way and thus breaking one of the main purposes of running these federated nodes.

Reply:
Thanks, I'll add that.

Just 2 outstanding gaps to be filled: 

1. where the Bitcoin Core configuration is actually specified (the bisq-network/btcnode repo seems empty)
2. any details of the "secure environment" full nodes need to run in.

Both these items are from the `Step 2. Set it Up` section.